.dockerignore

```diff
@@ -0,0 +1,25 @@
+**/.classpath
```

Comment:
Maybe reversing the list here would be better? 

What worked for me in the past is to ignore all with `*` and then explicitly add things we needed in the container with `!`

This makes the list an `include` which is easier to read & maintain

Reply:
This was auto-generated by VS for me so I haven't event looked at it really